dev/tests/integration/phpunit.xml.dist

```diff
@@ -21,8 +21,12 @@
         </testsuite>
         <testsuite name="Magento Integration Tests">
             <directory suffix="Test.php">testsuite</directory>
+            <directory suffix="Test.php">../../../app/code/*/*/Test/Integration</directory>
```

Comment:
Need to remove all mentions about MSI from phpunit.xml.dist (the same for WebApi)

Reply:
@naydav but there is no mentioning of MSI in this test configuration

Reply:
@naydav removed